src/config/config.js

```diff
@@ -106,6 +106,11 @@ export default {
    * @property {Boolean} rsaBlinding
    */
   rsaBlinding: true,
+  /**
+   * @memberof module:config
+   * @property {Number} rsaBitsMin Minimum RSA key size allowed for key generation
+   */
+  rsaBitsMin: 2048,
```

Comment:
Maybe `minRsaBits` or `minRSABits`? For consistency with `maxUseridLength` etc.
Or alternatively, `rsaGenerationMinBits`, in case we ever want to add a minimum for imported RSA keys.

Reply:
If we go with `minRSABits` (which I think makes sense in terms of casing) we should also make the other options and names consistent with this rule before v5 release (e.g. `maxUserIDLength`). Otherwise we should stick to `minRsaBits` imo.

Reply:
:+1:
(You changed it to `minRsaBits` everywhere except here I think)